backend/src/api/routes/admin-route.ts

```diff
@@ -30,13 +30,13 @@ router.get(
 
 router.post(
   '/users',
-  run((req) =>
-    registerCommonUsers(
+  run((req) => {
+    console.warn(req.body);
+    return registerCommonUsers(
       createDefaultUser(req.body.email),
       req.body.roleType,
-      req.companyId,
-    ),
-  ),
+    );
+  }),
```

Comment:
1. remove console.log
2. you cannot register menror/mentee user without companyId